rfcs/convergence/changelog-guidelines.md

```diff
@@ -0,0 +1,129 @@
+# Changelog (beachball changefile) guidelines
+
+## Summary
+
+This RFC proposes a set of guidelines when creating changefiles during a PR. These changefiles are directly translated
+(in most cases) to our changelog which is important for users to see what changed between releases of Fluent. Therefore
+this RFC proposes the following requirements for the changelog:
+
+- Changes that do not affect published code should not be in the changelog
+- Clear labels for change types
+- The detail of changes should be descriptive first and short second
+- Messages should be written for changes specific to the package, not the PR
+
+The changefiles also directly influence how packages are bumped and released in the release pipeline.
+
+## Problem statement
+
+The current v9 changelog contains (or contained before removal) the following bad patterns that should not be in the changelog:
+
+### Changelog entries that do not affect published code
+
+Changes such as:
+
+- stories are updated
+- dev dependencies are updated
+- tests were added/updated
+- scripts were modified
+
+Should not be included in the changelog. Generally when these kinds of changes are in the changelog they should be
+removed as soon as they are noticed.
```

Comment:
nit, splitting "changes such as"/"should not..." is bugging me--but up to you whether to change it or not
```suggestion
Certain types of changes don't affect published code and should not be in the changelog. For example:

- stories are updated
- dev dependencies are updated
- tests were added/updated
- scripts were modified

Generally when these kinds of changes are in the changelog they should be
removed as soon as they are noticed.
```